lib/template/config/initializers/log.rb

```diff
@@ -1,2 +1,3 @@
 Pliny.default_context = {}
 Pliny.default_context[:app] = Config.app_name if Config.app_name
+Pliny.default_context[:deployment] = Config.deployment
```

Comment:
Intentionally setting this outside the `default_context` initialization to keep the key ordering of log entries as

```
app=my-app deployment=development
```